<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShapeDialogClass</class>
 <widget class="QDialog" name="ShapeDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>580</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>810</width>
    <height>580</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>810</width>
    <height>580</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ShapeDialog</string>
  </property>
  <widget class="QLabel" name="steelNameLB">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>钢束名称：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="steelNameLE">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="steelPropertyLB">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>钢束特性值：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="steelPropertyCB">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="sigmaLB">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>张拉力：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sigmaLE">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="sigmaUnionLB">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>40</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>MPa</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="horCoordinateTW">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>320</width>
     <height>200</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>5</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>24</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>23</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>x(cm)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>y(cm)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>R(cm)</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="verCoordinateTW">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>320</width>
     <height>200</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>5</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>24</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>23</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>30</number>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>x(cm)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>z(cm)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>R(cm)</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="updateBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>530</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>更新图像</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmBtn">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>530</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>530</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
